### ISSUE TITLE
Bug Bash Feedback: Consistency issue with envs?

### ISSUE THREAD

**Issue by SoManyHs** [aws/amazon-ecs-cli-v2#375]
```
~/workspace/aws-reinvent-2018-trivia-game/static-site(master)$ archer env init
? What is your environment's name? staging

✔ Success! Environment staging already exists under project badgoose! Do nothing.

~/workspace/aws-reinvent-2018-trivia-game/static-site(master)$ archer env ls
test

~/workspace/aws-reinvent-2018-trivia-game/static-site(master)$ archer env init 
? What is your environment's name? stage

✔ Success! Environment stage already exists under project badgoose! Do nothing.
```
I started a new project under a new account, and it's saying I have envs aleady that i think exist only in a different account. I guess this is expected behavior, but it does mean that I can't actually list the envs or deploy to them? And it's also confusing bc if I didn't know better I'd think archer was punking me.

**Comment by kohidave** [comment]
This doesn't completely fix this but #395 will help.

**Comment by efekarakus** [comment]
Fixed with https://github.com/aws/amazon-ecs-cli-v2/pull/395
